.circleci/config.yml

```
@@ -1751,6 +1751,9 @@ workflows:
                 - master
           requires:
             - package-and-push-system-dev
+          matrix:
+            parameters:
+              max-dynamic-agents: [2]
```

it's a single parameter value so I suppose you don't really need the matrix?

```suggestion
          max-dynamic-agents: 2
```

---

yeah true, didn't give it much thought, sorry